pkgs/by-name/as/ast-grep/package.nix

```diff
@@ -12,24 +12,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "ast-grep";
-  version = "0.33.1";
+  version = "0.34.3";
 
   src = fetchFromGitHub {
     owner = "ast-grep";
     repo = "ast-grep";
     tag = version;
-    hash = "sha256-p7SJhkCoo4jBDyr+Z2+qxjUwWXWpVMuXd2/DDOM7Z/Q=";
+    hash = "sha256-r82BDCncRUSmIBQFwsrKDwKEKmvGm/lKtz1rYC47Ems=";
   };
 
-  cargoHash = "sha256-aCBEL+Jx4Kk7PWsxIgpdRdI7AnUAUEtRU4+JMxQ4Swk=";
+  cargoHash = "sha256-ro2RbXDf9TsU6WdI1tGzCix+xTJVxzN38vyVZonAuUg=";
 
   nativeBuildInputs = [ installShellFiles ];
 
-  # error: linker `aarch64-linux-gnu-gcc` not found
```

Comment:
@GaetanLepage Can you help me to see weather delete postPatch works in aarch64 linux.